### PR TITLE
Automated cherry pick of #2796: Remove duplicate Felix Health Port for Openshift

### PR DESCRIPTION
**Pull request by rene-dekker**
Cherry pick of #2796 on release-v1.31.

#2796: Remove duplicate Felix Health Port for Openshift

### CODE REVIEW

**Review by caseydavenport — APPROVED** [review]

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and delete the branch after I've merged it.

**Comment by marvin-tigera** [comment]
I'm sorry but I failed delete the branch after merging the pull request.

**Pull request merged**
